src/Vts/MonteCarlo/DataStructuresValidation/SimulationInputValidation.cs

```diff
@@ -166,24 +171,25 @@ private static ValidationResult ValidateCombinedInputParameters(SimulationInput
                         {
                             case true when
                                 ellipsoid.Center.X != 0.0 && ellipsoid.Center.Y != 0.0:

```

Comment:
I'm seeing an issue here with the floating point comparison, it suggests using a range:
https://www.jetbrains.com/help/resharper/CompareOfFloatsByEqualityOperator.html

Reply:
I understand jetbrains concern.  In this particular case, I truly am comparing with 0.0 though.  If it is slightly off, ellipsoid will be off center.  Since it is a user input (not determined from code), I would prefer to keep this as is.